02-Logging/01-Displaying-Status-Information.ipynb

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
<p>In the overview of the section you talk about "backends", I would suggest using the word <code>Writer</code>  there as well for consistency.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Good catch!</p>